src/m365/spo/commands/navigation/navigation-node-set.ts

```diff
@@ -42,7 +45,8 @@ class SpoNavigationNodeSetCommand extends SpoCommand {
         title: typeof args.options.title !== 'undefined',
         url: typeof args.options.url !== 'undefined',
         isExternal: typeof args.options.isExternal !== 'undefined',
-        audienceIds: typeof args.options.audienceIds !== 'undefined'
+        audienceIds: typeof args.options.audienceIds !== 'undefined',
+        openInNewWindow: typeof args.options.openInNewWindow !== 'undefined'
```

Comment:
@MathijsVerbeeck I just noticed also that this (I think) should also be added to the validation condition 
```typescript
// We should extend this condition
if (args.options.audienceIds === undefined && !args.options.url && args.options.isExternal === undefined && !args.options.title) { 
    return `Please specify atleast one property to update.`;
}
```

I think now it is impossible to update only `openInNewWindow` 🤔